Component/PetList/PetList.js

```diff
@@ -1,20 +1,48 @@
 import styled from "styled-components";
-import pets from "@/Lib/Data";
 import Pet from "../Pet/Pet";
+import Link from "next/link";
 
 const StyledPetList = styled.ul`
   display: flex;
   flex-direction: column;
   align-items: center;
   width: 100vw;
+  margin: 5% 0;
+
+  a {
+    display: grid;
+    place-content: center;
+    width: 25vw;
+    height: 10vh;
+    max-width: 200px;
+    font-size: 20px;
+    border-radius: 10px;
+    margin: 0 20px;
+    box-shadow: 2px 2px black;
+    cursor: pointer;
+    background-color: var(--signal-color);
+    background-image: var(--button-image);
+    color: #000000;
+    text-decoration: none;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &:active {
+      background-color: var(--secondary-color);
+    }
```

Comment:
do I also have to avoid nested styles inside for example a button or Link component? I created a Link component but as I have to put the Link element inside another element (as I cannot declare a "styled.Link") I don't now how I could avoid nesting the styles for the Link element inside its parent.
Also, do pseudo classes count as nested classes? If yes, how can I implement these features without nesting?

Reply:
You can extend the Link component using Styled Components:

```js
const StyledLink = styled(Link)``;
```

Pseudo classes don't count as nested classes. They need to be part of a parent, so that there is a reference.

Reply:
Ok cool, that helped a lot! 😇